schema/create_table_historic_data.sql

```diff
@@ -0,0 +1,14 @@
+CREATE DATABASE delays;
+
+\c delays
```

Comment:
what is this? :D

Reply:
changes the selected database to "delays" instead of the default Postgres.

Reply:
One question though: Can we create multiple databases that way? I think the Postgres image only allows creation of one database.

Reply:
Or do we save the monthly data in the same database?

Reply:
Let's just focus on one month for now

Reply:
No I mean the timetable data that your crawler is gathering. If we create the database using compose/kubernetes we can only create one database per container. So we would need to save both of our data in the same database. But that would be fine for now in my opinion.

Reply:
>  So we would need to save both of our data in the same database.

👍